tests/expected/c/10085-infinite-loops.c

```diff
@@ -0,0 +1,83 @@
+int main(void)
+{
+	int x;
+
+	do
+	{
+	}while(1);
+
+	do
+	{
+	}while(1);
+
+	do
+	{
+	}
+	while(1);
+
+	do
+	{
+	}while(1);
+
+	do
+	{
+	}
+	while (1);
+
+	do
+		x++; while(1);
```

Comment:
I think we could add a new like before ```while(1)``` when replacing into do-while loop. That should be an easy change and probably what everyone would expect, regardless of what settings is used for other options.